sdk/tables/azure-data-tables/azure/data/tables/_constants.py

```diff
@@ -5,6 +5,7 @@
 # --------------------------------------------------------------------------
 import platform
 import sys
+from ._generated.version import VERSION
```

Comment:
don't think you should import version from generated code since this appears to be multiapi and you've only generated one version. See [text analytics](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_base_client.py#L13) for inspiration (sidenote: this api version class is also exposed to the user through the init file)